packages/appstore/src/components/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { Button, Text, Icon, ProgressBarOnboarding } from '@deriv/components';
+
+type TOnboardingProps = {
+    contents: Record<
+        string,
+        {
+            component: React.ReactNode;
+            footer_header: string;
+            footer_text: string;
+            next_content?: string;
+            has_next_content: boolean;
+        }
+    >;
+};
+
+const Onboarding = ({ contents }: TOnboardingProps) => {
+    const amount_of_steps = Object.keys(contents);
+
+    const [step, setStep] = React.useState<number>(1);
+
+    const prevStep = () => {
+        if (step > 1) setStep(step - 1);
+    };
+
+    const nextStep = () => {
+        if (step < amount_of_steps.length) setStep(step + 1);
+    };
+
+    const onboarding_step = amount_of_steps[step - 1];
+
+    return (
+        <div className='onboarding-wrapper'>
+            <div className='onboarding-header'>
+                <Icon icon='IcWalletDeriv' height={96} width={128} />
+                <Icon icon='IcCross' custom_color='#fff' className='onboarding-header__cross-icon' />
+            </div>
+            <div className='onboarding-body'>
+                <Text as='h2' weight='bold' size='s' align='center' color='white'>
```

Comment:
`s` is the default value, no need to set the value
```suggestion
                <Text as='h2' weight='bold' align='center' color='white'>
```